00-open-a-shapefile.Rmd

```diff
@@ -159,31 +155,26 @@ aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 ```
 
 After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name.
+This can be done simple by entering the file name. Alternatively, we can use the `attributes()` function.
```

Comment:
is this a function or a method, @jsta 

Maybe we include this as a note?

    NOTE: we can also use the `attributes()` method to view objects attributes
    {: .notice }

if we add the .notice tag, this text will appear in a nice little notes box!